lib/iris/fileformats/nimrod.py

```diff
@@ -48,7 +48,7 @@
     "proj_biaxial_ellipsoid",
     "ensemble_member",
     "spare1",
```

Comment:
The nimrod documentation says this element is the origin model ID. I guess this is being ignored cos the information is coming from element 106 describing the source instead.

Reply:
Renamed